calculus/integration-partial-fractions.md

```diff
@@ -1 +1,117 @@
 # Partial fraction decomposition
+## Shortcut: Heaviside Cover-up Method
+
+$$ 
+\frac{3x+11}{(x-3)(x+2)} = \frac{A_1}{x-3} + \frac{A_2}{x+2} 
+$$
+
+Multiply by a factor and take limit:
+
+$$
+\lim_{x\to 3} \left( \frac{3x+11}{x+2} = A_1 + \frac{A_2(x-3)}{x+2} \right) 
+$$
+
+$$
+\frac{3(3)+11}{3+2} = A_1 
+$$
+
+$$
+ A_1 = 4 
+$$
+
+$$ 
+\lim_{x\to -2} \left( \frac{3x+11}{x-3} = \frac{A_1(x+2)}{x-3} + A_2 \right) 
+$$
+
+$$ 
+\frac{3(-2)+11}{-2-3} = A_2 
+$$
+
+$$
+A_2 = -1 
+$$
+
```

Comment:
An explanation of the steps in the method (cover up the factor, substitute) and its limitations would be useful here, perhaps put inside a `{topic}` directive.